### PR TITLE
[MSUE-51]

### PR DESCRIPTION
**Pull request by Toji-Thomas**
## Purpose

Add Validation APIs (send,resend,check)

## Summary

Added wrappers for the following APIs

- verification/send
- verification/resend
- verification/check

Added unit test for the above API wrappers(spec/unit/client_validationapi_spec.rb)

Added an example which makes real calls(examples/validation_apis.rb)
## Testing

Send(Real call)
send
Resend(Real call)
resend
Check(Real call)
Unit Test
unittest

## Checklist
- [X] The change was thoroughly tested manually
- [X] The change was covered with unit tests
- [X] The change was tested with real API calls (if applicable)
- [ ] Necessary changes were made in the integration tests (if applicable)
- [ ] New functionality is reflected in README

### CODE REVIEW

**Review by binish-exalture — APPROVED** [review]
The branch seems good. Approving...

**Review by viaskal-sift — APPROVED** [review]

File: lib/sift/client.rb [hunk 1]
```diff
@@ -720,6 +720,71 @@ def apply_decision!(configs = {})
...
 
+    def build_default_headers_post(api_key)
+      { 
+        "Authorization" => "Basic #{Base64.encode64(api_key+":")}",
+        "User-Agent" => "SiftScience/v#{@version} sift-ruby/#{VERSION}",
+        "Content-Type" => "application/json"
+      }      
+    end
+
+    def send(properties = {}, opts = {})
```
Comment (viaskal-sift) [inline lib/sift/client.rb:731]: the name of the method looks too generic, let's rename it to `verification_send`

File: lib/sift/client.rb [hunk 2]
```diff
@@ -720,6 +720,71 @@ def apply_decision!(configs = {})
...
+        :body => MultiJson.dump(delete_nils(properties)),
+        :headers => build_default_headers_post(api_key)
+      }
+      options.merge!(:timeout => timeout) unless timeout.nil?
+
+      response = self.class.post(Sift.verification_api_send_path(@version), options)
+      Response.new(response.body, response.code, response.response)
+    end
+
+    def resend(properties = {}, opts = {})
```
Comment (viaskal-sift) [inline lib/sift/client.rb:750]: the name of the method looks too generic, let's rename it to `verification_resend`

File: lib/sift/client.rb [hunk 3]
```diff
@@ -720,6 +720,71 @@ def apply_decision!(configs = {})
...
+        :body => MultiJson.dump(delete_nils(properties)),
+        :headers => build_default_headers_post(api_key)
+      }
+      options.merge!(:timeout => timeout) unless timeout.nil?
+
+      response = self.class.post(Sift.verification_api_resend_path(@version), options)
+      Response.new(response.body, response.code, response.response)
+    end
+
+    def check(properties = {}, opts = {})
```
Comment (viaskal-sift) [inline lib/sift/client.rb:769]: the name of the method looks too generic, let's rename it to `verification_check`

**Pull request merged**